cmd/agent/app/integrations.go

```diff
@@ -516,14 +516,14 @@ func parseWheelPackageName(wheelPath string) (string, error) {
 			for scanner.Scan() {
 				line := scanner.Text()
 
-				exp, err := regexp.Compile(wheelPackageName)
+				exp, err := regexp.Compile(wheelPackageNamePattern)
```

Comment:
Why are we compiling the regex for every line? Shouldn't this be outside of the scanner for loop?

EDIT: actually outside of both nested for loops.

Reply:
👍 fixed.

1) Changed all regex as global vars. 
2) Use of `regexp.MustCompile`

@ofek Do you mind reviewing this part?